esp/bindings/http/platform/httptransport.cpp

```diff
@@ -798,6 +798,49 @@ void CHttpMessage::logSOAPMessage(const char* message, const char* prefix)
     return;
 }
 
+static const char* POST_METHOD_STR = "POST ";
+static bool skipLogContent(const char* httpHeader)
+{
+    if (!startsWith(httpHeader, POST_METHOD_STR))
+        return false;
+
+    const char* servicePtr = httpHeader + 5;
+    if (isEmptyString(servicePtr) || (servicePtr[0] != '/'))
+        return false;
+
+    const char* methodPtr = strchr(++servicePtr, '/');
+    if (!methodPtr)
+        return false;
+
+    unsigned serviceType = 0;
+    if (startsWithIgnoreCase(servicePtr, "ws_access/"))
+        serviceType = 1;
+    else if (startsWithIgnoreCase(servicePtr, "ws_account/"))
+        serviceType = 2;
+    if (serviceType == 0)
+        return false;
+
+    StringBuffer espMethod;
+    const char* tail = strchr(++methodPtr, '.');
+    if (tail && (startsWithIgnoreCase(tail, ".xml") || startsWithIgnoreCase(tail, ".json")))
```

Comment:
This code seems slightly unusual.  A request of

POST /ws_access/UserResetPass.xml

is not logged?  A separate question is should this be logged, but without the details?

Reply:
@ghalliday about the ".xml/.json" part.  For ESP you can add those modifiers to any method name to specify the content type of the response.  Although it's mean primarily for HTTP GET as with HTTP POST the content type defaults to the request content type and is also affected by the "Accept" header.

But for this code he is basically ignoring a content type specification.

Reply:
I don't know who or why this logging was completely blocked.  Could consider logging some but not all info.

Reply:
Should analyse the possible contents of those requests and decide which items should be blocked. Will be in: https://track.hpccsystems.com/browse/HPCC-30519.